gamemodes/terrortown/entities/weapons/weapon_zm_carry.lua

```diff
@@ -205,13 +188,13 @@ function SWEP:Reset(keep_velocity)
 			phys:EnableMotion(true)
 		end
 
-		if not keep_velocity and (not GetGlobalBool("ttt_prop_throwing") or self.EntHolding:GetClass() == "prop_ragdoll") then
-			KillVelocity(self.EntHolding)
+		if not keep_velocity  then
+			KillVelocity(ctarget)
 		end
 	end
 
-	self:SetCarriedRag(nil)
-	self.EntHolding = nil
+	self:SetCarryType(CARRY_TYPE_NONE)
+	self:SetCarryTarget(NULL)
```

Comment:
```suggestion
	self:SetCarryTarget(nil)
```

Reply:
![image](https://github.com/TTT-2/TTT2/assets/5711436/c5a49f11-979e-4d5b-b58e-44c34a27857d)
![image](https://github.com/TTT-2/TTT2/assets/5711436/244acc5c-cc64-4133-89d9-02399adb810f)
causes a lot of network noise cause Reset keeps writing a nil to a value that can only contain NULL, compare to normal conditions:
![image](https://github.com/TTT-2/TTT2/assets/5711436/3e4bf88f-6d0b-4194-a1e9-6a39e8333b08)
![image](https://github.com/TTT-2/TTT2/assets/5711436/f7bb2abc-56f4-4428-a7dc-f1ab18cb9459)